src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -1030,6 +1030,37 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
             'value': frontend
         })
 
+        if spec.rgw_user_counters_cache:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_user_counters_cache',
+                'value': str(spec.rgw_user_counters_cache).lower(),
+            })
+        if spec.rgw_bucket_counters_cache:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_bucket_counters_cache',
+                'value': str(spec.rgw_bucket_counters_cache).lower(),
+            })
+
+        if spec.rgw_user_counters_cache_size:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_user_counters_cache_size',
+                'value': str(spec.rgw_user_counters_cache_size),
+            })
+
+        if spec.rgw_bucket_counters_cache_size:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
```

Comment:
we're generating `utils.name_to_config_section(daemon_spec.name())` 5 times in this `prepare_create` function (1 from before this PR, 4 being added).  Can we just have this stored in a variable before the first use case and just reference the variable 5 times instead?